pkg/api/vlabs/validate.go

```diff
@@ -111,6 +111,13 @@ func (o *OrchestratorProfile) Validate(isUpdate bool) error {
 						}
 					}
 				}
+				if helpers.IsTrueBoolPointer(o.KubernetesConfig.EnablePodSecurityPolicy) {
+					if o.KubernetesConfig.EnableRbac != nil {
```

Comment:
We could use `helpers.IsTrueBoolPointer` again here (in this case we'd use it to determine not true). E.g.:

`if helpers.IsTrueBoolPointer(o.KubernetesConfig.EnablePodSecurityPolicy) && !helpers.IsTrueBoolPointer(o.KubernetesConfig.EnableRbac)`

Reply:
updated